applications/nrf_desktop/doc/ble_state.rst

```diff
@@ -3,4 +3,82 @@
 BLE state module
 ################
 
-The BLE state module is used to control Zephyr BLE connection callbacks and send the required ``ble_peer_event``.
+Use the ``ble_state`` module to:
```

Comment:
I am not sure if we can call these `scenarions`, I think it would be refer to it as `functionalities`. The Event Manager events are send when handling Zephyr Bluetooth callbacks.